src/app/settings.py

```diff
@@ -143,3 +144,74 @@
 
 AUTH_USER_MODEL = "users.User"
 
+# VersatileImageField
+
+VERSATILEIMAGEFIELD_SETTINGS = {
+    # The amount of time, in seconds, that references to created images
+    # should be stored in the cache. Defaults to `2592000` (30 days)
+    'cache_length': 2592000,
+    # The name of the cache you'd like `django-versatileimagefield` to use.
+    # Defaults to 'versatileimagefield_cache'. If no cache exists with the name
+    # provided, the 'default' cache will be used instead.
+    'cache_name': 'versatileimagefield_cache',
+    # The save quality of modified JPEG images. More info here:
+    # https://pillow.readthedocs.io/en/latest/handbook/image-file-formats.html#jpeg
+    # Defaults to 70
+    'jpeg_resize_quality': 70,
+    # The name of the top-level folder within storage classes to save all
+    # sized images. Defaults to '__sized__'
+    'sized_directory_name': '__sized__',
+    # The name of the directory to save all filtered images within.
+    # Defaults to '__filtered__':
+    'filtered_directory_name': '__filtered__',
+    # The name of the directory to save placeholder images within.
+    # Defaults to '__placeholder__':
+    'placeholder_directory_name': '__placeholder__',
+    # Whether or not to create new images on-the-fly. Set this to `False` for
+    # speedy performance but don't forget to 'pre-warm' to ensure they're
+    # created and available at the appropriate URL.
+    'create_images_on_demand': True,
+    # A dot-notated python path string to a function that processes sized
+    # image keys. Typically used to md5-ify the 'image key' portion of the
+    # filename, giving each a uniform length.
+    # `django-versatileimagefield` ships with two post processors:
+    # 1. 'versatileimagefield.processors.md5' Returns a full length (32 char)
+    #    md5 hash of `image_key`.
+    # 2. 'versatileimagefield.processors.md5_16' Returns the first 16 chars
+    #    of the 32 character md5 hash of `image_key`.
+    # By default, image_keys are unprocessed. To write your own processor,
+    # just define a function (that can be imported from your project's
+    # python path) that takes a single argument, `image_key` and returns
+    # a string.
+    'image_key_post_processor': None,
+    # Whether to create progressive JPEGs. Read more about progressive JPEGs
+    # here: https://optimus.io/support/progressive-jpeg/
+    'progressive_jpeg': False
+}
+
+# A dictionary used to specify ‘Rendition Key Sets’ that are used for both serialization
+# or as a way to ‘warm’ image files so they don’t need to be created on demand (i.e.
+# when settings.VERSATILEIMAGEFIELD_SETTINGS['create_images_on_demand'] is set to False)
+# which will greatly improve the overall performance of your app. Here’s an example:
+
+VERSATILEIMAGEFIELD_RENDITION_KEY_SETS = {
+    'image_gallery': [
+        ('gallery_large', 'crop__800x450'),
```

Comment:
The `crop` will change the ratio of images! Don't you think is better to have something more general and using thumbnail for the boilerplate?

Something like: 
```
VERSATILEIMAGEFIELD_RENDITION_KEY_SETS = {
    "default_sizes": [
        ("full_size", "url"),
        ("medium", "thumbnail__400x400"),
        ("small", "thumbnail__100x100"),
        ("tiny", "thumbnail__50x50"),
    ]
}
```

Then we can create a more specific groups as we need on wich project! That's, of course, is just an opinion.

Reply:
I think that the current configuration is more for an example that a default config for all projects 🤔 
Maybe we can leave `VERSATILEIMAGEFIELD_RENDITION_KEY_SETS ` params commented, like:
```
VERSATILEIMAGEFIELD_RENDITION_KEY_SETS = {
 #   'image_gallery': [
 #     ('gallery_large', 'crop__800x450'),
 ...
 }
```
What do you think?

Reply:
Yep, I just copied those values from the tutorial page... Maybe we can comment it out to force this decision.
I'm just fearful that people might not even notice these settings, it might be a good idea to include them on the project's README!

Reply:
Nice, hehe but I can't denied that I prefer to have a default, the dev who will be in the new project will just deal with that if he needs a different configuration.
About your fear, I completely agree with you and having a default will mitigate this problem.